lib/vtls/wolfssl.c

```diff
@@ -327,6 +368,26 @@ wolfssl_connect_step1(struct Curl_easy *data, struct connectdata *conn,
     infof(data, "Cipher selection: %s", ciphers);
   }
 
+  curves = SSL_CONN_CONFIG(curves);
```

Comment:
So these names are all "curves" and not "tls13-ciphers" ?

I'm asking because I don't really understand the difference...

Reply:
These are TLS 1.3 "groups". In the parlance of TLS 1.2, they were called "curves" . These do not specify cipher suites because in TLS 1.3 cipher suites specify the symmetric encryption while the "groups" specify the key exchange algorithm.